sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/tests/Microsoft.OpenTelemetry.Exporter.AzureMonitor.Integration.Tests/TelemetryItemTests.cs

```diff
@@ -137,19 +135,87 @@ private TelemetryItem RunLoggerTest(Action<ILogger<TelemetryItemTests>> testScen
                     .AddProcessor(processor));
             });
 
-            // ACT
             using var serviceProvider = serviceCollection.BuildServiceProvider();
             var logger = serviceProvider.GetRequiredService<ILogger<TelemetryItemTests>>();
+
+            // ACT
             testScenario(logger);
 
             // CLEANUP
             processor.ForceFlush();
-            //Task.Delay(100).Wait(); //TODO: HOW TO REMOVE THIS WAIT?
 
             Assert.True(transmitter.TelemetryItems.Any(), "test project did not capture telemetry");
             return transmitter.TelemetryItems.Single();
         }
 
-        // TODO: INCLUDE ADDITIONAL TESTS VALIDATING ILOGGER + ACTIVITY
+        /// <summary>
+        /// This test is to verify that the correlation Ids from Activity are set on Log Telemetry.
+        /// This test will generate two telemetry items and their Ids are expected to match.
+        /// This test does not inspect any other fields.
+        /// </summary>
+        [Fact]
+        public void VerifyLoggerWithActivity()
+        {
+            // SETUP
+            var ActivitySourceName = "MyCompany.MyProduct.MyLibrary";
+            using var activitySource = new ActivitySource(ActivitySourceName);
+
+            var transmitter = new MockTransmitter();
+
+            var processor1 = new BatchExportProcessor<Activity>(new AzureMonitorTraceExporter(
+                options: new AzureMonitorExporterOptions
+                {
+                    ConnectionString = EmptyConnectionString,
+                },
+                transmitter: transmitter));
+
+            var processor2 = new BatchExportProcessor<LogRecord>(new AzureMonitorLogExporter(
+                options: new AzureMonitorExporterOptions
+                {
+                    ConnectionString = EmptyConnectionString,
+                },
+                transmitter: transmitter));
+
+            Sdk.CreateTracerProviderBuilder()
+                .SetSampler(new AlwaysOnSampler())
+                .AddSource(ActivitySourceName)
+                .AddProcessor(processor1)
+                .Build();
+
+            var serviceCollection = new ServiceCollection().AddLogging(builder =>
+            {
+                builder.SetMinimumLevel(LogLevel.Trace)
+                    .AddOpenTelemetry(options => options
+                    .AddProcessor(processor2));
```

Comment:
The indentation here is a bit weird?

Reply:
AddProcessor is not a sibling of AddOpenTelemetry, consider adding indentation or putting it on the same line with AddOpenTelemetry.